src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/tasks/ShadowJar.kt

```diff
@@ -165,10 +164,6 @@ public abstract class ShadowJar :
   @Input // Trigger task executions after excludes changed.
   override fun getExcludes(): MutableSet<String> = super.getExcludes()
 
-  override fun manifest(configureClosure: Closure<*>): ShadowJar = apply {
-    super.manifest(configureClosure)
-  }
```

Comment:
This seems useless, and I don't want to expose `Closure` as the public API again.